package.json

```diff
@@ -13,6 +13,9 @@
   "scripts": {
     "test": "mocha --ui tdd --reporter spec test/*.js"
   },
+  "files": [
```

Comment:
The “files” field is dangerous - the only way published files should ever be restricted is by npmignore.